README.md

```diff
@@ -1 +1,66 @@
-# Portfolio-setup-and-mobile-firs
\ No newline at end of file
+# Portfolio-setup-and-mobile-firs
+# Project Name
+
+This project is to develop the two sections of a Portfolio. It is mainly meant to setup the portfolio. 
+Setting up HTML/CSS linter to the repository, adding Github workflows, creating .gitignore file and adding files to be ignored in committing are the intial tasks done to develop this initial version of the two sections of the Portfolio. Then after, Figma is used to extract images and some CSS attributes and values. Therefore, the hero section of the Portfolio web application for mobile devices is developed in this project.
+
+
+## Built With
+
+- Major languages: HTML, CSS
+- No Frameworks used
+- Technologies used: VS code, Git, Github, linters, Github workflows
+
+## Live Demo (if available)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/amare1990)
+- email address: amaremek@gmail.com
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you add an MIT license to your root folder of your project so that this link directs users to the License because currently this link is broken and doesn't link anywhere.